README.md

```diff
@@ -84,4 +84,30 @@ If the default `SymbolProvider` is missing useful information for the language /
 
 ## `AutocompleteManager` API
 
-The `AutocompleteManager` is exposed as a [provided service](http://flight-manual.atom.io/behind-atom/sections/interacting-with-other-packages-via-services/). Its main public method is `watchEditor` which allows external editors to register for autocompletions from providers with a given set of labels.
+The `AutocompleteManager` is exposed as a [provided service](http://flight-manual.atom.io/behind-atom/sections/interacting-with-other-packages-via-services/).
+The service, named `autocomplete.manager`, provides a function that can execute a callback on the `AutocompleteManager` object.
+External packages can access this service with the following code.
+
+In `package.json`:
+```
+{
+  "consumedServices": {
+    "autocomplete.manager": {
+      "versions": {
+        "1.0.0": "consumeAutocompleteManager",
+      }
+    }
+  }
+}
+```
+In the main module file:
+```
+consumeAutocompleteManager(service) {
```

Comment:
Okay, I can give that a try! The only performance reason that I can imagine is that the autocomplete manager creates the providers which might have to do some indexing.